static/src/js/components/GridCheckboxPanel/GridCheckboxPanel.jsx

```diff
@@ -1,13 +1,118 @@
-import React from 'react'
-import Row from 'react-bootstrap/Row'
-import Col from 'react-bootstrap/Col'
-
-const GridCheckboxPanel = () => (
-  <Row>
-    <Col>
-      This is where the Checkbox Panel would go.
-    </Col>
-  </Row>
-)
+import React, { useState } from 'react'
+import { kebabCase } from 'lodash-es'
+import { cloneDeep } from '@apollo/client/utilities'
+import PropTypes from 'prop-types'
+import removeEmpty from '../../utils/removeEmpty'
+
+// eslint-disable-next-line import/no-cycle
+import GridLayout from '../GridLayout/GridLayout'
+
+const GridCheckboxPanel = ({
+  layoutGridSchema,
```

Comment:
I think this property should be called `layout` now (lot of them were renamed in the transition)